### DISCUSSION TITLE
Osu!lazer not starting

### DISCUSSION THREAD

**Discussion by yt6983138** [ppy/osu#27651] [Q&A] [answered]
### Type

Crash to desktop

## installer
if i run the installer, the icon shows and the progress bar progresses, then the window just died
## game
if i launch the game i can see the process just open up for a moment then died in task manager, no log are produced (%appdata%/osu folder not created also), in event viewer (pic attached below) shows that it encounters a connection issue, i have tested a bit more:
1. osu stable: working
2. user debug version: working
3. user **release** version: just dies
4. release from website: dies again
now i have no idea why this happens

oh btw, i wonder is installer project included there? i dont see it

### Screenshots or videos

![image](https://github.com/ppy/osu/assets/83499886/03d48f76-4c40-48a3-b59f-f4d1a9a90a3e)
![image](https://github.com/ppy/osu/assets/83499886/6923c30d-9d99-4d28-9ecd-692b09354b1b)
(no log found in %appdata%/osu)

### Version

2024.312.0

### Logs

(no log found at %appdata%/osu, folder not exist)
[Report.zip](https://github.com/ppy/osu/files/14625511/Report.zip)

**Comment by cdwcgt** [comment]
can you check port 44823 is bind to other process?
and you can try to launch tournament, use `osu.exe --tournament`

**Reply by yt6983138** [reply]
44823 is not binded by tcp nor udp
yes i can launch in tournament normally, any idea?

**Reply by cdwcgt** [reply]
lazer won't start when 44823(the IPC port) was binded. that is, multiple simultaneous starts are not allowed.
and your lazer may tried to pass some IPC message which cause connected refused.

Sorry for replying now, I did not receive the message notification

**Comment by yt6983138** [comment, marked as answer]
huh about 2 weeks later (now) it just started working again...? 🤔

**Reply by Rekunan** [reply]
Does the event viewer and everything look fine?

**Reply by cdwcgt** [reply]
maybe just the port was released?

**Comment by pytheor** [comment]
Hi, I came across this same problem that the game will crash on startup because it fails to bind on 44823. In my case the port was unused (checked through TCPView) and even restart didn't help. According to this post [https://stackoverflow.com/questions/48478869/cannot-bind-to-some-ports-due-to-permission-denied](https://stackoverflow.com/questions/48478869/cannot-bind-to-some-ports-due-to-permission-denied) it's caused by Hyper-V/WSL. Restarting _winnat_ service fixed it.
This probably only affects a small portion of users but it might be worth considering retrying different ports of one fails.

**Reply by yt6983138** [reply]
btw sometimes you need to preserve the ports yourself to prevent them being binded by other service again

i found the reason after months later doing asp.net project (unable to bind port but not used), apparently asp.net has more developers than osu so i found the solution pretty quickly
